### ISSUE TITLE
Fix missing alt attributes in MediaAsides on news pages

### ISSUE THREAD

**Issue by soshwosh** [lsst-epo/rubin-obs-client#1139]
Some of the most visited news pages on the main site have redundant alt text for images in the media asides, lowering our ADA compliance scores. Do some light discovery on the NOIRLab media ingesting pattern to determine whether the issue lies in our code or if it's an external issue (alt text not retrievable or not set on NOIRLab side) and add the alt values if possible.

**Comment by soshwosh** [comment]
Unfortunately the[ NOIRLab API schema](https://noirlab.edu/public/documentation/schema) does not provide alt text for the images. As a solution, we [use the title of the image for alt text](https://github.com/lsst-epo/rubin-obs-client/blob/f1cfeb6c8a52942c6bfbaa200095c16afd18535c/helpers/noirlab.ts#L71). Unless alt text is added to the API response on the NOIRLAB side, this is the only solution for now.
